proposed-decisions.md

```diff
@@ -0,0 +1,128 @@
+
+## Revenue recognition
+[Related issue 12](https://github.com/stichtingsem/functional-overview/issues/12)
+
+[Related issue 13](https://github.com/stichtingsem/functional-overview/issues/13)
+
+Revenue recognition is influenced by tax legislation, competitive propositions and commercial contracts/agreements. Partners in the ecosystem should determine their own approach to revenue recognition, but the services should ensure valid & transparent transactions and enable various transaction models
+
+## Identity
+[Related Issue 14](https://github.com/stichtingsem/functional-overview/issues/14)
+
+- Product: EAN
+- User: ECK-ID
+- School: Digi Delivery ID (RIO will be evaluated on applicability)
+
+## Bring SIS (LAS) and LML in sync
+[Related issue 15](https://github.com/stichtingsem/functional-overview/issues/15)
+
+SIS (LMS) data will be synced with LML
+
+Boundaries:
+- Target binding based on role
+- School is in control of the accessibility to the entities/data within SIS
+
+Preparation phase:
+- Subjects
+- Courses
+
+Execution phase:
+- Forecast (base for entitlement)
+
+## First Time Right
+[Related Issue 16](https://github.com/stichtingsem/functional-overview/issues/16)
+
+First Time Right will **not** be a principle, but there are (additional) requirements:
+- Avoidance of errors and repairs
+    - Determine when data is up to date and ready for use (data quality)
+- Updates (LML) will be shared (near) real time
+    - It is a dynamic process and updates will prevent errors (and repairs)
+    
+## In Scope POC
+[Related Issue 17](https://github.com/stichtingsem/functional-overview/issues/17)
+
+### In Scope:
+- VO (secondary) and MBO (vocational)
+- Full Digital (100% digital)
+- Blended (combi’s, LiFo)
+- Folio (incl. paper vouchers)
+    - Partners (SA, MP) within the eco-system have to evaluate the impact of the proposed solutions on their supply chain (forecasts/prognosis, purchase, stock, etc)
+- ECK
+
+[Related Issue 18](https://github.com/stichtingsem/functional-overview/issues/18)
+
+### Out of scope (for now):
+- Setup & Support
+    - Most important next step in the SEM pilot
+    - More decentralized = more risk > How to reduce risk?
+- Communication 
+- School processes / administration (SIS, ELO, IDP)
+    - Not enrolled students (0e Jaars)
+- Teacher licenses (including trial licenses)
+    - Additional services for teachers > prepare access locations for teacher licenses
+- Kennisnet ARP administration
+- Progress & Results (Part of Use Case 4.0)
+- Data covenant 
+- Non happy flows
+- Returns
+    - Return reasons: Changes LML (school), Mutations SIS (change of subjects, OKO
+- NON ECK
+
+## What is Fulfilment Update LMS
+[Related Issue 19](https://github.com/stichtingsem/functional-overview/issues/19)
+
+Update contains contains information about order confirmations, forecasts and availability
+
+## Usage Information blended learning
+[Related Issue](https://github.com/stichtingsem/functional-overview/issues/20)
+
```

Comment:
To clarify: it wouldn't matter if it's a blended or pure digital product

Reply:
I related this to the original issue, but usage information will always (full digital and blended) be communicated in the ecosystem